lib/maths/CTimeSeriesDecomposition.cc

```diff
@@ -249,20 +249,18 @@ bool CTimeSeriesDecomposition::addPoint(core_t::TTime time,
 bool CTimeSeriesDecomposition::applyChange(core_t::TTime time,
                                            double value,
                                            const SChangeDescription& change) {
+    m_PeriodicityTest.maybeClear(time, change.s_Magnitude[0]);
```

Comment:
`maybeClear` has been removed, so I suspect a merge conflict.

Reply:
Yes, hit it. I just removed this line can just go in the backport as well.